packages/core/package.json

```diff
@@ -35,21 +35,22 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.45",
+    "@testing-library/react-hooks": "^5.0.3",
```

Comment:
Oh I missed the build errors. I believe you need to lower this a lot, unfortunately. The rest of the project is on `"^3.4.2"` and until we have time to bump it in a coordinated fashion I think you will have to choose that too

Reply:
(and remember to commit the yarn.lock changes as well)